cmd/cluster/resize/controlplane_node.go

```diff
@@ -72,9 +72,9 @@ func newCmdResizeControlPlane() *cobra.Command {
   The user will be prompted to send a service log after the resize is complete.`,
 		Example: `
   # Resize all control plane instances to m5.4xlarge using control plane machine sets
-  osdctl cluster resize control-plane -c "${CLUSTER_ID}" --machine-type m5.4xlarge --reason "${OHSS}" --cpms
+  osdctl cluster resize control-plane -c "${CLUSTER_ID}" --machine-type m5.4xlarge --reason "${OHSS}"
 
-  # Resize a control plane node to m5.4xlarge, should be repeated for all control plane nodes
+	# Legacy: Resize a control plane node on a cluster without active controlplane machineset, should be repeated for all control plane nodes
```

Comment:
Unsure if this is just GitHub but looks like some extra whitespace in front of this heading

Reply:
mhh yeah that looks weird :D let me see

Reply:
fixed